scenarios/proxy.benchmarks.yml

```diff
@@ -51,11 +51,17 @@ jobs:
       BranchOrCommit: master
       DockerFile: docker/envoy/envoy.dockerfile
       DockerImageName: envoy-reverseproxy
-      DockerContextDirectory: docker/envoy
+      DockerContextDirectory: .
```

Comment:
I don't think this works. The context directory should be docker/envoy or the relative path in the docker file won't match

Reply:
Hmm. I changed it because noticed [the same change in NGINX scenario](https://github.com/aspnet/Benchmarks/blob/b0e6602510ef5196dc51532a750e7706e9d4d222/scenarios/proxy.benchmarks.yml#L38). Is it a bug?

Reply:
Yes, that's a problem. I must have changed it when I did local tests. I am fixing it in nginx. Please do the same here

Reply:
Fixed